pilot/pkg/networking/core/v1alpha3/envoyfilter/util_test.go

```diff
@@ -94,28 +93,19 @@ func TestReplaceAndInsert(t *testing.T) {
 		t.Run(c.name, func(t *testing.T) {
 			// replaceFunc
 			got, applied := replaceFunc(slices.Clone(c.input), f)
-			if !reflect.DeepEqual(c.replace, got) {
+			if !slices.Equal(c.replace, got) {
 				t.Errorf("replaceFunc: want %+v but got %+v", c.replace, got)
 			}
-			if !reflect.DeepEqual(c.applied, applied) {
+			if c.applied != applied {
 				t.Errorf("replaceFunc: want %+v but got %+v", c.applied, applied)
 			}
 
 			// insertBeforeFunc
 			got, applied = insertBeforeFunc(slices.Clone(c.input), f)
-			if !reflect.DeepEqual(c.insertBefore, got) {
+			if !slices.Equal(c.insertBefore, got) {
 				t.Errorf("insertBeforeFunc: want %+v but got %+v", c.insertBefore, got)
 			}
-			if !reflect.DeepEqual(c.applied, applied) {
-				t.Errorf("insertBeforeFunc: want %+v but got %+v", c.applied, applied)
-			}
-
-			// insertAfterFunc
-			got, applied = insertAfterFunc(slices.Clone(c.input), f)
```

Comment:
Thanks, the `insertAfterFunc` test was removed. Was it deleted accidentally?

Reply:
No. I will fix it